### ISSUE TITLE
Attribute error on fetching live trade data and detailed company data

### ISSUE THREAD

**Issue by CaffeineDuck** [Samrid-Pandit/nepse-api#17]
### Error:
```py
  File "/home/acer/.local/lib/python3.9/site-packages/nepse/security/core.py", line 115, in get_company_live_price
    (await self._client_wrapper.get_json(BASE_LIVE_TRADE_URL)).get("content")
AttributeError: 'list' object has no attribute 'get'
```

### How to reproduce:
~~Try getting the data from the wrapper about a company after the market is closed.~~

### Solution:
~~Create a `MarketClosed` exception that inherits the `NepseException` and create a decorator to check if the market has closed or not.~~

PS: The above-mentioned problem and solution aren't valid, as shown in the later updates of this issue. Scroll down for more info.

**Comment by CaffeineDuck** [comment]
### Update: 
The issue was caused due to NEPSE changing their API from the `GET` request method, to the `POST` request method. I am trying to find out which APIs were affected and trying to get out a patch as soon as possible.

**Comment by CaffeineDuck** [comment]
### Workaround for now:
Don't use the `get_company_detailed`, `get_company_live_price` and `get_companies_live_price`, instead use `get_company` and `get_companies`. Sorry for the inconvinence.

**Comment by CaffeineDuck** [comment]
Fixed in #18
